web/impact/impact/tests/test_method_override_middleware.py

```diff
@@ -0,0 +1,24 @@
+# MIT License
+# Copyright (c) 2017 MassChallenge, Inc.
+
+from django.urls import reverse
+
+from impact.middleware.method_override_middleware import METHOD_OVERRIDE_HEADER
+from impact.tests.api_test_case import APITestCase
+from impact.tests.contexts import UserContext
+
+
+class TestMethodOverrideMiddleware(APITestCase):
+
+    def test_patch(self):
+        context = UserContext()
+        user = context.user
+        with self.login(username=self.basic_user().username):
```

Comment:
I would rename this test to 'test_patch_as_post' and would also add another test that tests setting HTTP_X_HTTP_METHOD_OVERRIDE to GET

Reply:
@shanbady You mean a test that shows it failing if a GET is passed in?